tests/test_integration.py

```
@@ -1,8 +1,15 @@
 # tests/test_integration.py
 
+from unittest.mock import MagicMock
 import pytest
-from forecast import forecast, forecastapi
-from model import model
+from forecast import forecast, model
+
+@pytest.fixture
+def mock_model(mocker):
+    mock_model = MagicMock()
+    mock_model.predict.return_value = np.array([[42]])
```

Ruff

Undefined name `np`

(F821)